src/libraries/System.Private.CoreLib/src/System/Runtime/InteropServices/Marshal.cs

```diff
@@ -677,10 +677,10 @@ public static unsafe IntPtr StringToHGlobalAnsi(string? s)
                 throw new ArgumentOutOfRangeException(nameof(s));
```

Comment:
Nit: For consistency throughout this file, this change be changed to `ThrowHelper.ThrowArgumentOutOfRangeException(ExceptionArgument.s)`